geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -861,6 +861,15 @@
       "region-time-to-live-expiration-action";
   public static final String CREATE_REGION__REGIONEXPIRATIONTTLACTION__HELP =
       "Action to be taken on a region that has exceeded the TTL expiration.";
+  public static final String CREATE_REGION__EVICTION_ACTION = "eviction-action";
+  public static final String CREATE_REGION__EVICTION_ACTION__HELP =
+      "The eviction action to apply. Must be either 'destroy' or 'overflow-to-disk'";
```

Comment:
The EvictionAction.stringTable defined is "local-destroy", not "destroy". COmmnet also applies to the CREATE_REGION__MSG__INVALID_EVICTION_ACTION string.

Reply:
That is correct. The story calls for the eviction action to be `destroy` which is how it is exposed to the user via gfsh. The internal naming remains unchanged - i.e. `local-destroy`. I will double-check to ensure this is the correct requirement.

Reply:
Yes, you're right. The current implementation does have the action as `local-destroy`. I have another PR in the wings which switches `EvictionAction` to an enum and that was what I was thinking of. When that hits, the eviction action option will be `destroy`.